### PR TITLE
Update mac-jamfpro-policies.md

### PR DESCRIPTION
**Pull request by msbemba**
updated the navigation steps

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6948

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/mac-jamfpro-policies.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ You'll need to take the following steps:
 
    ![Image of WindowsDefenderATPOnboarding file.](images/plist-onboarding-file.png)
 
-2. In the Jamf Pro dashboard, select **New**.
+2. Login to Jamf Pro and navigate t0 **Computers** > **Configuration Profiles** and click **New**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mac-jamfpro-policies.md]:
```suggestion
2. Sign in to Jamf Pro, navigate to **Computers** > **Configuration Profiles**, and select **New**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**